site/src/_pages/docs/0_index.mdx

```diff
@@ -1,12 +1,26 @@
 ## Overview
 
-These pages provide a practical introduction to implementing the block protocol, whether you are:
+The Block Protocol is a global standard for building interactive blocks connected to structured data.
 
-- interested in [developing blocks](https://blockprotocol.org/docs/developing-blocks), or
-- interested in [embedding blocks in your application](https://blockprotocol.org/docs/embedding-blocks)
+The following pages provide a practical introduction for developers who want to:
+
+- [develop blocks](https://blockprotocol.org/docs/developing-blocks) and publish them to the block registry
+- [embed blocks](https://blockprotocol.org/docs/embedding-blocks) in their own application
 
 For the full technical details, read [the Block Protocol specification](https://blockprotocol.org/spec).
 
+### What's a block?
+
+Blocks are the basic units we use to build pages on the modern web. They are rectangular, full-width sections stacked on top of one another – also known as "block-level" elements.
+
```

Comment:
```suggestion

A block is any valid HTML published to the [Block Hub](https://blockprotocol.org/hub).

```

cc @CiaranMn @MaggieAppleton per our 'quickest start guide' @ https://blockprotocol.org/docs/developing-blocks

Reply:
Rephrased this but tried to keep the same gist in the "What is a Block Protocol block" section